wire-library/wire-tests-swift/src/main/swift/AllTypes.swift

```diff
@@ -1943,6 +1948,11 @@ extension AllTypes : Hashable {
 }
 #endif
 
+#if swift(>=5.5) && !WIRE_REMOVE_SENDABLE
+extension AllTypes : @unchecked Sendable {
```

Comment:
I explicitly did not make `Heap<>` conform to sendable since that would necessarily be @unchecked.
Due to that, this need @unchecked conformance.

In theory, we could change things to be:
```swift
struct HeapCow<T> {
    private var storage: Heap<T>
    public var wrappedValue {
        get { storage.wrappedValue }
        set { 
            copyStorage()
            storage.wrappedValue = newValue
        }
    }
}
```

and then remove `copyStorage()` from the heap-allocated protos

Reply:
If we did this, then `HeapCow` could be `@unchecked Sendable`

Reply:
I agree with what you have. `Heap` itself is not `Sendable` and marking it as such isn't correct. Put another way, the thing that makes these (unchecked) `Sendable` is the calls to `copyStorage`, which are in the proto object code.